eoscostarica/eosio-spend-explorer
# Feat/configure hyperion endpoint for token transfer tracking 49

### Feat/configure hyperion endpoint for token transfer tracking 49

### What does this PR do?

- Resolve #49 
- Resolve #68 
- Resolve #67 
- Resolve #76 

### Steps to test

1. Run Project
2. See localhost:9695
3. See Data->eden_election for historic elections

#### CheckList
